theme/static/css/hydroshare_core.css

```diff
@@ -1785,6 +1799,9 @@ span.name-label {
   }
 }}
 
+#items {
+	overflow: auto;
```

Comment:
@mauriel133 you can covert all tabs into spaces using PyCharm to ensure consistency